plugins/statistics.koplugin/main.lua

```diff
@@ -240,6 +240,12 @@ end
 
 -- Reset the (volatile) stats on page count changes (e.g., after a font size update)
 function ReaderStatistics:onUpdateToc()
+    -- Note: this is called *after* a onPageUpdate() with the new current_page (showing
```

Comment:
`curr_page`?

Reply:
(The whole comment block could do with a grammar pass, because it's a tad clunky and confusing right now :s).